cmd/kubelet/app/bootstrap.go

```diff
@@ -74,7 +73,7 @@ func bootstrapClientCert(kubeconfigPath string, bootstrapPath string, certDir st
 	if err != nil {
 		return fmt.Errorf("unable to build bootstrap key path: %v", err)
 	}
-	keyData, generatedKeyFile, err := loadOrGenerateKeyFile(keyPath)
+	keyData, generatedKeyFile, err := certutil.LoadOrGenerateKeyFile(keyPath)
```

Comment:
I'm not clear what you mean. `cmd/kubelet/app/BUILD` already has this line: 

```
        "//vendor:k8s.io/client-go/util/cert",
```

and `staging/src/k8s.io/client-go/util/cert/io.go` appears to have all the imports needed by the new function.